source/MaterialXCore/Document.h

```diff
@@ -502,6 +519,12 @@ class Document : public Element
     /// @name Callbacks
     /// @{
 
+    /// Enable all observer notifications		
+    virtual void enableNotifications() { }
```

Comment:
Following up on the suggestion below, I would name these methods "onSuspendCallbacks" and "onRestoreCallbacks", and place them at the end of the list, after "onBeginUpdate" and "onEndUpdate".

Reply:
As stated in my general comment, these methods are not needed anymore.

Reply:
I will make this change. See my general comment below.